packages/maskbook/src/social-network/ui.ts

```diff
@@ -71,6 +71,7 @@ export async function activateSocialNetworkUI(): Promise<void> {
     ui.collecting.profilesCollector?.(signal)
     ui.injection.pageInspector?.(signal)
     if (Flags.toolbar_enabled) ui.injection.toolbar?.(signal)
+    ui.injection.toolBoxInNavBar?.(signal)
```

Comment:
Should we remove `ui.injection.toolbar`?

Reply:
Temporarily not.

Reply:
should it in a flag?

Reply:
oh, it requires flags rather than here but for some of its entries,  updated.

Reply:
update